OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
I don't understand why a Muscle would not appear red by default if a Ligament is green by default. It seems that the mechanism should be consistent either way.